zebra/zebra_mroute.c

```diff
@@ -37,22 +37,34 @@ void zebra_ipmr_route_stats(ZAPI_HANDLER_ARGS)
 	struct mcast_route_data mroute;
 	struct stream *s;
 	int suc = -1;
+	int size = 0;
 
 	memset(&mroute, 0, sizeof(mroute));
-	STREAM_GET(&mroute.src.u.prefix4, msg, 4);
-	STREAM_GET(&mroute.grp.u.prefix4, msg, 4);
-	STREAM_GETL(msg, mroute.ifindex);
+	STREAM_GETL(msg, mroute.family);
```

Comment:
Please modify the pimd code to pass down a v4 family only with this commit.  Else when someone needs to do a git bisect against pim code, this will just plain break.

Reply:
Thanks Donald for the review. I have squashed the commits.